src/explorer/WebAppProvider.ts

```diff
@@ -51,7 +53,10 @@ export class WebAppProvider implements IChildProvider {
     }
 
     public async createChild(node: IAzureNode<IAzureTreeItem>, showCreatingNode: (label: string) => void, actionContext: IActionContext): Promise<IAzureTreeItem> {
-        const newSite: Site | undefined = await createWebApp(util.getOutputChannel(), node.ui, actionContext, node.credentials, node.subscriptionId, node.subscriptionDisplayName, showCreatingNode);
+        // get workspace fspath here, but only if one is opened
```

Comment:
Can you explain this comment? It doesn't make sense to me

Reply:
Ah, crap, that was a remnant from my old PR when I was retrieving the workspace in the extension and passing it to createWebApp.